packages/flutter_driver/lib/src/native/android.dart

```diff
@@ -20,44 +21,57 @@ final class AndroidNativeDriver implements NativeDriver {
   /// The [tempDirectory] argument can be used to specify a custom directory
   /// where the driver will store temporary files. If not provided, a temporary
   /// directory will be created in the system's temporary directory.
+  ///
+  /// @nodoc
```

Comment:
what is nodoc?

Reply:
Good question! It makes this constructor not show up in dartdoc.